deploy/kubernetes/kibana/kibana.gcloud.yaml

```diff
@@ -26,6 +26,8 @@ spec:
       labels:
         name: kibana
         deployment: {{ DEPLOY_TO }}
+    spec:
+      automountServiceAccountToken: true
```

Comment:
what does this do?

Reply:
This was documented here: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-service-mesh-linkerd.html#k8s-service-mesh-linkerd-kibana-apm

linkerd refused to mesh the kibana service until I added that. I'm not entirely sure if that has any effect if ECK is off in another cluster, but worth taking another look at it as we move elasticsearch back into the seqr cluster.